### PR TITLE
Adding new optional parameter to display or hide banner.

### PR DESCRIPTION
**Pull request by Judreu**
To avoid an instability, by default the banner is displayed New parameter name is: bool displayBanner

### CODE REVIEW

**Comment by Judreu** [comment]
Closed by misunderstanding of GitHub sorry, the pull request is sill available !

**Comment by subhamayd2** [comment]
The point of this package is the Day Night animation. You want to hide that itself? :(

**Comment by Judreu** [comment]
Hello, in some cases, I need to gain space to make all widgets fit in the available space. 
To be homogeneous, I just want to display the animation only where there is enough space.

**Comment by subhamayd2** [comment]
Oh okay. Sounds good. :)

**Comment by subhamayd2** [comment]
@all-contributors please add @AnlienApp  for code

File: example/lib/main.dart [hunk 1]
```diff
@@ -100,6 +100,7 @@ class _HomeState extends State<Home> {
                   minHour: 9,
                   maxHour: 21,
                   is24HrFormat: false,
+                  displayBanner: false
```
Comment (subhamayd2) [inline example/lib/main.dart]: Please remove this

File: lib/lib/daynight_timepicker.dart [hunk 2]
```diff
@@ -151,6 +151,7 @@ dynamic showPicker({
   sunrise = const TimeOfDay(hour: 6, minute: 0),
   sunset = const TimeOfDay(hour: 18, minute: 0),
   duskSpanInMinutes = 120,
+  bool displayBanner = true,
```
Comment (subhamayd2) [inline lib/lib/daynight_timepicker.dart:156]: Please add this in the doc comment above

**Review by subhamayd2 — CHANGES_REQUESTED** [review]

File: example/lib/main.dart (hunk 1 above)
Comment (Judreu) [reply example/lib/main.dart]: Sorry for this fail, I forget to remove my test modification :(

**Comment by Judreu** [comment]
I published new modification to fit your recommendation 
Hope it will be fine with that

**Comment by Judreu** [comment]
Hello, i don't know if I have to do something more, this is my first pull request for another project.

**Comment by subhamayd2** [comment]
This looks good. Thanks for the help ❤️

**Comment by subhamayd2** [comment]
Hey, I noticed we already had a prob called `displayHeader`, which basically hides the banner. I think that's what you want. I should have checked the code first before merging this, but anyway, I have reverted this.

**Pull request merged**